autocertkit/cpu_tests.py

```diff
@@ -82,7 +82,7 @@ def _create_test_threads(self, session, vm_ref_list):
             threads.append(create_test_thread(lambda: TimeoutFunction(ssh_command(vm_ip,
                                                                                   self.username,
                                                                                   self.password,
-                                                                                  self.cmd_str),
+                                                                                  self.cmd_str)["stdout"],
```

Comment:
In implement of cpu_test.py, it does not care the output (including both stdout and stderr from self.cmd_str) of ssh_command() at all, even if TimeoutFunction() return it. What test is interested in is files /usr/lib/lmbench/results/i686-pc-linux-gnu/* and fetched by call plugin retrieve_lmbench_logs
```
        # retrieve the logs
        log.debug("%s test is complete, retrieving logs" % self.test)
        res = self._call_plugin(session, vm_ref_list,
                                'retrieve_' + self.test + '_logs')
```

You know I refactored ssh_command() to let it return dict with exit code, stdout and stderr separated, here I add ["stdout"] is just to let TimeoutFunction() return a string as before, rather than dict, because I checked other places of using TimeoutFunction() they all return string currently.

So needless to append ["stderr"] here.

Reply:
👍